scripts/install/requestrr.sh

```diff
@@ -0,0 +1,178 @@
+#!/bin/bash
+# Requestrr installation
+# Author: Brett
+# Copyright (C) 2021 Swizzin
+# Licensed under GNU General Public License v3.0 GPL-3 (in short)
+#
+#   You may copy, distribute and modify the software as long as you track
+#   changes/dates in source files. Any modifications to our software
+#   including (via compiler) GPL-licensed code must also be made available
+#   under the GPL along with build & install instructions.
+#shellcheck source="sources/functions/os"
+. /etc/swizzin/sources/functions/os
+
+user=$(cut -d: -f1 < /root/.master.info)
+
+echo_progress_start "Downloading source files"
+case "$(_os_arch)" in
+    "amd64") wget -qO "/tmp/requestrr.zip" "$(curl -sNL https://api.github.com/repos/darkalfx/requestrr/releases/latest | grep -Po 'ht(.*)linux-x64(.*)zip')" >> ${log} 2>&1 ;;
+    "armhf") wget -qO "/tmp/requestrr.zip" "$(curl -sNL https://api.github.com/repos/darkalfx/requestrr/releases/latest | grep -Po 'ht(.*)linux-arm(.*)zip')" >> ${log} 2>&1 ;;
+    "arm64") wget -qO "/tmp/requestrr.zip" "$(curl -sNL https://api.github.com/repos/darkalfx/requeAstrr/releases/latest | grep -Po 'ht(.*)linux-arm64(.*)zip')" >> ${log} 2>&1 ;;
+    *)
+        echo_error "Arch not supported"
+        exit 1
+        ;;
+esac
+echo_progress_done "Source downloaded"
+
+echo_progress_start "Extracting archive"
+unzip /tmp/requestrr.zip -d /opt/ >> "$log" 2>&1
+rm /tmp/requestrr.zip
+mv requestrr* requestrr
+echo_progress_done "Archive extracted"
+
+touch /install/.requestrr.lock
+
+echo_progress_start "Creating requestrr user and setting permssions"
+useradd -M --shell=/bin/false requestrr
```

Comment:
why not use `--system` and set the homedir to the folder in opt? not sure what `-M`

Reply:
will be changed to ``useradd --system -d /opt/requestrr requestrr``